### PR TITLE
JSON.yaml: Add JSON definition

### PR DESCRIPTION
**Pull request by vex8**

### CODE REVIEW

File: data/Language/JSON.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+line_continuation: ''
+delimiters:
+  # From coala JSON language definition
+  # JSON don't have any comments
+  # String delimiter:
+  - double_quote
+  # Block delimiter:
+  - curly_braces
+  - square_braces
+keywords: []
```
Comment (Man-Jain) [inline data/Language/JSON.yaml]: You can remove this line as there are no keywords.

File: data/Language/JSON.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+identifier: JSON
+wikidata: Q2063
+full_name: JavaScript Object Notation
+extensions: 
+  - json
+line_continuation: ''
+delimiters:
+  # From coala JSON language definition
```
Comment (Man-Jain) [inline data/Language/JSON.yaml]: No need for this comment

File: data/Language/JSON.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  # JSON don't have any comments
```
Comment (Man-Jain) [inline data/Language/JSON.yaml]: This one too is not needed.

File: data/Language/JSON.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  # String delimiter:
+  - double_quote
```
Comment (jayvdb) [inline data/Language/JSON.yaml]: JSON string have an escape character. Choose a more specific delimiter

File: data/Language/JSON.yaml (hunk 1 above)
Comment (vex8) [reply data/Language/JSON.yaml]: I was unsure about what to do with that field, I'll remove it now

File: data/Language/JSON.yaml (hunk 4 above)
Comment (vex8) [reply data/Language/JSON.yaml]: Oh, that's right. I was a bit confused by what delimiter to use, but I'll check them

Comment (jayvdb) [reply data/Language/JSON.yaml]: Yup, JSON uses backslash escaping.

File: data/Language/JSON.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+identifier: JSON
+wikidata: Q2063
+full_name: JavaScript Object Notation
+extensions: 
+  - json
+aliases:
+  - json
+line_continuation: ''
```
Comment (jayvdb) [inline data/Language/JSON.yaml]: What does this mean?

Does JSON have a line continuation token?
If not, remove this property.

Comment (vex8) [reply data/Language/JSON.yaml]: Alright, so we just drop the properties if it's not used?

File: git add . [hunk 6]
```diff
@@ -0,0 +1,14 @@
+identifier: JSON
```
Comment (jayvdb) [inline git add .]: Please remove this file ;)

Comment (Man-Jain) [inline git add .]: This file is created by mistake. Remove it :smiley:

Comment (vex8) [reply git add .]: uh oh. I'm sorry, I'll remove it now!

Comment (vex8) [reply git add .]: Oops, I thought it wasn't there since git status doesn't show this file

Comment (Man-Jain) [reply git add .]: Np :smile:

**Comment by vex8** [comment]
Thanks @jayvdb and @Man-Jain for reviewing my first PR! I sure make a lot of mistake for one, but I'll try to not do as much in the future.

**Comment by jayvdb** [comment]
ack https://github.com/coala/coAST/pull/10/commits/d0771c65248d38b0114fdfd72ddd272da6ce2dc3

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated fastforward with [GitMate.io](https://gitmate.io) was successful! :tada:

**Pull request merged**